src/tests/ftest/nvme/nvme_object.py

```diff
@@ -194,6 +195,7 @@ def test_nvme_object_multiple_pools(self):
         threads = []
         index = 0
         for size in self.pool_size[:-1]:
+            time.sleep(1)
```

Comment:
Are we sure 1 second is enough? Why we added this 1 second delay? Hope we will not land in some intermittent failure.

Reply:
Added so threads have some time in between before it begins.